### ISSUE TITLE
Curser focus on two places before of end of line and typing problem in angular json editor.

### ISSUE THREAD

**Issue by PParshive** [josdejong/jsoneditor#1182]
I am facing typing problem in **angular json editor** in my project and curser focus on two places before the end of line that why typing problem create while typing in a json editor. Please find the solution as soon as possible.
"ang-jsoneditor": "^1.10.3",
"jsoneditor": "5.19.2"
Mention version i am using in my project.

**Comment by josdejong** [comment]
Thanks for reporting @PParshive . 

Can you try if this is still an issue in the latest version of jsoneditor? The version you're using, `5.19.2`, is very old (date `2018-08-02`).

**Comment by PParshive** [comment]
Yes, I tried with latest version "9.1.4", still getting same issue. I am referring this example https://stackblitz.com/edit/ang-jsoneditor?file=package.json. Example showing proper output but getting problem in my application.

**Comment by josdejong** [comment]
> Example showing proper output but getting problem in my application.

Hm, so how can we reproduce this issue?
